### PR TITLE
fix: added comments to improve readability

### PR DESCRIPTION
**Pull request by satwikrprabhu**
Also removed some unnecessary console.log() statements

### CODE REVIEW

File: src/components/ItemInfo.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const ItemInfo = () => {
           <InfinitySpin width="200" color="#424242" />
         </div>
       ) : (
-        <div className="overflow-hidden flex-col flex items-center h-screen justify-center bg-gray-800 ">
+        <div className="overflow-hidden flex-col flex items-center h-screen justify-center bg-gray-800 ">\
```
Comment (RishabhJain0721) [inline src/components/ItemInfo.js]: I guess slash is placed by mistake
@satwikrprabhu

**Comment by satwikrprabhu** [comment]
sorry for that. fixed it

File: src/components/ItemInfo.js (hunk 1 above)
Comment (satwikrprabhu) [reply src/components/ItemInfo.js]: fixed it

**Comment by RishabhJain0721** [comment]
Thanks @satwikrprabhu 
It would be great if you could give this repo a star.

**Pull request merged**